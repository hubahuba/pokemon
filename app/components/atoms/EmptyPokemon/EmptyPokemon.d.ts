export interface EmptyPokemonProps {
  message: string;
}
