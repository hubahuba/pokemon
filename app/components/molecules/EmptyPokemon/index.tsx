import * as React from 'react';
import {ImageBackground, Text} from 'react-native';
import {EmptyPokemonProps} from '~molecules/EmptyPokemon/EmptyPokemon';

export default function EmptyPokemon({message}: EmptyPokemonProps) {
  return (
    <ImageBackground
      className="min-h-screen items-center px-4"
      resizeMode="contain"
      source={require('~/assets/images/sad.jpg')}>
      <Text className="font-[Raleway-SemiBold] text-[28px] mt-[170] text-slate-600 text-center">
        {message}
      </Text>
    </ImageBackground>
  );
}
