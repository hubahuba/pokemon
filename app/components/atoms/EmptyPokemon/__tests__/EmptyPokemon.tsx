import React from 'react';
import renderer from 'react-test-renderer';
import EmptyPokemon from '../index';

it('renders Card correctly', () => {
  const tree = renderer.create(<EmptyPokemon message="testing" />).toJSON();
  expect(tree).toMatchSnapshot();
});
