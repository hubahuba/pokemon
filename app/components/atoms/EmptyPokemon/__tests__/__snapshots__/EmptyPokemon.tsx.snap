// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Card correctly 1`] = `
<View
  accessibilityIgnoresInvertColors={true}
  style={
    [
      {
        "minHeight": 1334,
      },
      {
        "alignItems": "center",
      },
      {
        "paddingLeft": 16,
        "paddingRight": 16,
      },
    ]
  }
>
  <Image
    resizeMode="contain"
    source={
      {
        "testUri": "../../../app/assets/images/sad.jpg",
      }
    }
    style={
      [
        {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        {
          "height": undefined,
          "width": undefined,
        },
        undefined,
      ]
    }
  />
  <Text
    style={
      [
        {
          "fontFamily": "Raleway-SemiBold",
        },
        {
          "fontSize": 28,
        },
        {
          "marginTop": 170,
        },
        {
          "color": "#475569",
        },
        {
          "textAlign": "center",
        },
      ]
    }
  >
    testing
  </Text>
</View>
`;
